### PR TITLE
(Chore) Edits response for update function

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[PT-Link](https://www.pivotaltracker.com/story/show/170912713)

- Edits response for update function

### CODE REVIEW

File: app/controllers/api/v1/admin/articles_controller.rb [hunk 1]
```diff
@@ -14,8 +14,9 @@ def create
 
   def update
     authorize(current_user)
-
-    Article.update(params[:id], published: params[:article][:published], publisher_id: current_user.id)
+    Article.update(params[:id], published: params[:published], publisher_id: current_user.id)
```
Comment (tochman) [inline app/controllers/api/v1/admin/articles_controller.rb]: Hmm... I'm really not onboard with why you choose to perform this action first and THEN pull the article from the database. Why not do this in one step?

File: spec/requests/api/admin/update_spec.rb [hunk 2]
```diff
@@ -26,9 +24,7 @@
       before do
         patch "/api/v1/admin/articles/1",
         params: {
-          article: {
```
Comment (tochman) [inline spec/requests/api/admin/update_spec.rb:29]: So are we NOT wrapping the params into an `artilcle` key?

**Review by tochman — COMMENTED** [review]
Not sure about the changes introduced in this one

File: app/controllers/api/v1/admin/articles_controller.rb [hunk 3]
```diff
@@ -14,8 +14,9 @@ def create
...
+    article = Article.find(params[:id])
+    render json: article
```
Comment (oliverochman) [inline app/controllers/api/v1/admin/articles_controller.rb]: why do you want to send back the article?

Comment (tochman) [reply app/controllers/api/v1/admin/articles_controller.rb]:
```
render head: :ok 
```

Comment (mantanquzzmo) [reply app/controllers/api/v1/admin/articles_controller.rb]: fixed

File: spec/requests/api/admin/update_spec.rb (hunk 2 above)
Comment (mantanquzzmo) [reply spec/requests/api/admin/update_spec.rb:29]: wrapping it into object now

File: app/controllers/api/v1/admin/articles_controller.rb (hunk 1 above)
Comment (mantanquzzmo) [reply app/controllers/api/v1/admin/articles_controller.rb]: Rendering ok response so no need to pull article

**Review by oliverochman — APPROVED** [review]

**Pull request merged**